scot/eegtopo/geo_spherical.py

```diff
@@ -41,6 +41,11 @@ def vector(self):
         """position in 3d space"""
         return self._pos3d
 
+    @property
+    def list(self):
```

Comment:
Yes.. should probably be named to_list(), and declared a function, not a property.
This is legacy code and should be dealt with in a different PR.